src/main/kotlin/godot/entrygenerator/EntryGenerator.kt

```diff
@@ -231,10 +231,10 @@ object EntryGenerator {
             .filter { ktClass ->
                 var containingFilePath = ktClass.containingKtFile.virtualFile.path
                 srcDirs.forEach { srcDir ->
-                    containingFilePath = containingFilePath.removePrefix(srcDir)
+                    containingFilePath = containingFilePath.removePrefix(srcDir.replace(File.separatorChar, '/'))
```

Comment:
Maybe add a comment here that we need this explicitly for windows. Otherwise i see myself changing this in the future because i will not remember this issue we had.